uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/FederatedLoginIT.java

```diff
@@ -0,0 +1,347 @@
+package org.cloudfoundry.identity.uaa.integration.feature;
+
```

Comment:
Please review and remove unused imports.

Reply:
Done